config/jobs/kubernetes-sigs/sig-windows/release-1.25-windows-presubmits.yaml

```diff
@@ -20,8 +20,7 @@ presubmits:
     extra_refs:
     - org: kubernetes-sigs
       repo: cluster-api-provider-azure
-      # TODO: update to release-1.10 when Go 1.20 is backported to kubekins-e2e.
-      base_ref: release-1.8
+      base_ref: release-1.10
```

Comment:
for release-1.10 we also need the Azure Workload identity preset everywhere

Reply:
Done.